lua/nvim-tree/git/cli.lua

```diff
@@ -0,0 +1,136 @@
+local notify = require "nvim-tree.notify"
+local log = require "nvim-tree.log"
+-- vim.loop got renamed in 0.10.0
+-- ref: https://github.com/neovim/neovim/pull/22846
+local uv = vim.loop or vim.uv
```

Comment:
Please use with the current pattern of directly calling `vim.loop.xxx` instead of adding a local.

Reply:
@alex-courtis This follows the above style of calling global imports and ensures nvim-cross compatibility. Should I fallback to likely breaking in the future `vim.loop`?

Reply:
Please follow the existing code style.

We will deal with that change when we change to a minimum version of vim that provides `vim.uv`

Reply:
Nitpicking.